.pre-commit-config.yaml

```diff
@@ -226,3 +201,11 @@ repos:
             uvicorn,
             jsbeautifier,
           ]
+  - repo: local
+    hooks:
+      - id: sphinx-lint
+        name: sphinx-lint
+        language: python
+        entry: sphinx-lint
+        additional_dependencies: [sphinx-lint]
+        types: [rst]
```

Comment:
In addition to the linter, isn't it also better to check for broken links using the [`linkchecker`](https://www.writethedocs.org/guide/tools/testing/#link-testing)? Or it serves a better purpose in the CI/CD pipeline?